concent_api/core/tasks.py

```diff
@@ -106,6 +126,18 @@ def upload_finished(subtask_id: str) -> None:
         Subtask.SubtaskState.ACCEPTED,
         Subtask.SubtaskState.FAILED
     ]:
+        delete_deposit_claim(
```

Comment:
You should not touch the claims here. It's a duplicate notification. If the verification timed out, succeeded or failed, the claims must have already been processed. If the verification is still going, the claims should stay as is and wait for the result.

Reply:
:+1: done